apiserver/facades/client/sshclient/shim.go

```diff
@@ -68,12 +82,13 @@ func (m *sshMachine) AllDeviceSpaceAddresses(ctx context.Context) (network.Space
 type backend struct {
 	*state.State
 
-	networkService NetworkService
+	networkService     NetworkService
+	applicationService ApplicationService
 }
 
 // GetMachineForEntity takes a machine or unit tag (as a string) and
 // returns the associated SSHMachine.
-func (b *backend) GetMachineForEntity(tagString string) (SSHMachine, error) {
+func (b *backend) GetMachineForEntity(ctx context.Context, tagString string) (SSHMachine, error) {
```

Comment:
Can this be lifted into the facade directly so we don't have to go via the backend shim.

OR at the very least, mark this is as deprecated and requires future changes.